### DISCUSSION TITLE
Welcome to RPGBattleEngine Discussions!

### DISCUSSION THREAD

**Discussion by HecknTarnation** [HecknTarnation/RPGBattleEngine#2] [General]
👋 Welcome!
We’re using Discussions as a place to connect with other members of our community. We hope that you:

Ask questions you’re wondering about.
Share ideas.
Engage with other community members.
Welcome others and are open-minded. Remember that this is a community we
build together 💪.

**Comment by gmilette** [comment]
This project is a great idea. Thanks for creating it!
